src/main/java/com/razorpay/Utils.java

```diff
@@ -48,6 +52,20 @@ public static boolean verifySignature(String payload, String expectedSignature,
     return isEqual(actualSignature.getBytes(), expectedSignature.getBytes());
   }
 
+  public static String generateOnboardingSignature(JSONObject attributes, String secret) {
```

Comment:
@bhavyay - this will also change according to new impl?

Reply:
also this has to be live secret

Reply:
When test client id is passed, test secret should be passed and live secret needs to be passed for live client id

Reply:
Updated the new implementation